<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Back Office</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Authentification</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelErreurIcon">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="pixmap">
            <pixmap resource="nWBackOffice.qrc">:/icones/mesIcones/inconWarning.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelErreurText">
           <property name="font">
            <font>
             <family>Monospace</family>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: red</string>
           </property>
           <property name="text">
            <string>Erreur de Saisie !</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelLogin">
           <property name="text">
            <string>Login*: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditLogin"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPassword">
           <property name="text">
            <string>Password*:  </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditPassword">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButtonValider">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&amp;Valider</string>
           </property>
           <property name="icon">
            <iconset resource="nWBackOffice.qrc">
             <normaloff>:/icones/valider</normaloff>:/icones/valider</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAnnuler">
           <property name="text">
            <string>&amp;Annuler</string>
           </property>
           <property name="icon">
            <iconset resource="nWBackOffice.qrc">
             <normaloff>:/icones/quitter</normaloff>:/icones/quitter</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="nWBackOffice.qrc"/>
 </resources>
 <connections/>
</ui>
